tools/jil/util/browser-matcher.js

```diff
@@ -413,12 +434,14 @@ features.es6 = new BrowserMatcher()
     new MatcherRule(TYPE_INCLUDE, 'firefox@>=29'),
     new MatcherRule(TYPE_INCLUDE, 'ios@>=16.0')
   ]);
+  features.sharedWorkersFull = features.sharedWorkers.and(features.supportESMWorkers);
+
   features.serviceWorkers = new BrowserMatcher([
     new MatcherRule(TYPE_EXCLUDE, '*@*'),
     new MatcherRule(TYPE_INCLUDE, 'chrome@>60'),  // v60 has unexpected behavior of working on unsecured HTTP tests
     new MatcherRule(TYPE_INCLUDE, 'edge@>=17'),
     new MatcherRule(TYPE_INCLUDE, 'safari@>=11.1'),
     new MatcherRule(TYPE_INCLUDE, 'firefox@>=44'),
-    new MatcherRule(TYPE_INCLUDE, 'android@>=105'),
+    new MatcherRule(TYPE_INCLUDE, 'android@>=106'),
```

Comment:
I realized late that there's no android version 100+, and instead there's an obscured mapping between chromium version to android version. 
I see this from Sauce: https://changelog.saucelabs.com/en/update-to-google-chrome-version-100-on-android-emulators
So basically the default we've used up to now (JWP) goes up to version 9.0 and the new "GoogleAPI emulator" (W3C) goes up to 13.0 now.
Can you change this to `@>=6.0` while you're at it? And the one for ESMWorkers above (line 424) can be changed to probably `@>=9.0` which should be a safe equivalent of `chrome@>=80`.